config/m4/gdrcopy.m4

```diff
@@ -38,20 +39,26 @@ AS_IF([test "x$with_gdrcopy" != "xno"],
                             gdrcopy_happy="no"])
             ], [gdrcopy_happy="no"])
 
+        AS_IF([test "x$gdrcopy_happy" = "xyes"],
+            [AC_CHECK_LIB([gdrapi], [gdr_copy_to_mapping],
+                [GDR_DEFINES="-DUCT_CUDA_GDR_COPY_HAS_COPY_TO_MAPPING"],
```

Comment:
GDR_DEFINES cppflags is not the way to do it
AC_CHECK_DECLS would produce HAVE_xx macro in config.h